packages/node_modules/@ciscospark/widget-recents/src/reducer.js

```diff
@@ -3,20 +3,33 @@ import mercury from '@ciscospark/redux-module-mercury';
 import user from '@ciscospark/redux-module-user';
 import spaces from '@ciscospark/redux-module-spaces';
 
-import {UPDATE_STATUS} from './actions';
+import {
+  UPDATE_STATUS,
+  STORE_INCOMING_CALL,
+  DELETE_INCOMING_CALL,
+  UPDATE_WEBRTC_SUPPORT
+} from './actions';
 
 export const initialState = fromJS({
+  incomingCall: null,
```

Comment:
Should we make it `incomingCalls`? What happens if we have multiple calls coming in at the same time? I know the UI will only display one, but if once we answer or decline the first one, we should display the second next.